modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPythonCodegen.java

```diff
@@ -1009,19 +997,27 @@ private ModelsMap postProcessModelsMap(ModelsMap objs) {
             model.getVendorExtensions().putIfAbsent("x-py-readonly", readOnlyFields);
 
             // import models one by one
-            if (!modelImports.isEmpty()) {
+            {
```

Comment:
why removed the check here?

Reply:
I don't remember, I will add it back.

I need to change the place where `x-py-model-imports` (there could be resources to imports even if there no dependencies between models), I will move it outside the `if` block.

Reply:
I re-added the condition and moved the `otherImports.exports()` imports out of the block.